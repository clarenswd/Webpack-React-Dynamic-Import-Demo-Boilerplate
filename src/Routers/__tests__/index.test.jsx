import React from 'react';
import renderer from 'react-test-renderer';
import Router from '../';

it('renders correctly', () => {
  const tree = renderer
    .create(<Router />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});