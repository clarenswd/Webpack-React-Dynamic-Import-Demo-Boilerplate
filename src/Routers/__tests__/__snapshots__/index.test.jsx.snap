// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <nav
    className="nav"
  >
    <ul>
      <li>
        <a
          aria-current="page"
          className="navItem navItemSelected"
          href="#/"
          onClick={[Function]}
          style={Object {}}
        >
          Main
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          className="navItem"
          href="#/leon/"
          onClick={[Function]}
        >
          Leon
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          className="navItem"
          href="#/claire/"
          onClick={[Function]}
        >
          Claire
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          className="navItem"
          href="#/chris/"
          onClick={[Function]}
        >
          Chris
        </a>
      </li>
    </ul>
  </nav>
  <h2>
    Choose the characters
  </h2>
</div>
`;
